src/paxcounter_orig.conf

```diff
@@ -29,6 +29,9 @@
 #define BLESCANWINDOW                   80      // [milliseconds] scan window, see below, 3 .. 10240, default 80ms
 #define BLESCANINTERVAL                 80      // [illiseconds] scan interval, see below, 3 .. 10240, default 80ms = 100% duty cycle
 
+// Use libpax instead of default counting algorithms
+#define LIBPAX                          0
+
 // Corona Exposure Notification Service(ENS) counter
 #define COUNT_ENS                       1       // count found number of devices which advertise Exposure Notification Service
```

Comment:
`#define COUNT_ENS 1` makes no sense in conjunction with `#define BLECOUNTER 0`

Reply:
You are right. We only changed this in the local paxcounter.conf. This is a not working configuration as you pointed out.
It looks like you already fixed it on the development branch.